1.8/tutorial/app1.md

```diff
@@ -0,0 +1,39 @@
+---
+post_title: Deploying first Application
+nav_title: Deploying first Application
+menu_order: 3
+---
+
+
+# Prerequisites
+In order to get started with this tutorial you should to have access to a running DC/OS OSS cluster and the DC/OS CLI installed and configured.
```

Comment:
`- Access to a DC/OS cluster.`
`- DC/OS CLI [installed](/docs/1.8/usage/cli/install/).`